bool is_valid
string frame_id
geometry_msgs/Point position
geometry_msgs/Quaternion orientation